README.md

```diff
@@ -117,60 +117,61 @@ The following code creates an element with the specified `tag`
 v(tag: string): HNode[];
 ```
 
-where `tag` is in the form: element.className(s)#id, e.g.
```

Comment:
👍 If its still _possible_ though, then maybe add to the reference guide (when written).